spec/content_block_tools/content_block_reference_spec.rb

```diff
@@ -87,19 +109,77 @@
             expect(result.count).to eq(3)
 
             expect(result[0].document_type).to eq("contact")
-            expect(result[0].content_id).to eq(contact_uuid)
+            expect(result[0].identifier).to eq(contact_uuid)
             expect(result[0].embed_code).to eq("{{embed:contact:#{contact_uuid}/example_field_1}}")
 
             expect(result[1].document_type).to eq("contact")
-            expect(result[1].content_id).to eq(contact_uuid)
+            expect(result[1].identifier).to eq(contact_uuid)
             expect(result[1].embed_code).to eq("{{embed:contact:#{contact_uuid}/example_field_2}}")
 
             expect(result[2].document_type).to eq("content_block_email_address")
-            expect(result[2].content_id).to eq(content_block_email_address_uuid)
+            expect(result[2].identifier).to eq(content_block_email_address_uuid)
             expect(result[2].embed_code).to eq("{{embed:content_block_email_address:#{content_block_email_address_uuid}/another_field}}")
           end
         end
       end
     end
+
+    describe "when there is embedded content with an alias" do
+      let(:contact_alias) { "contact-alias" }
```

Comment:
are we doing kebab-case then, as that's the normal slug mode? and I guess it differentiates with the field name as camel_case?

Reply:
The field names will normally be kebab case too. I'll update the tests to save confusion